Makefile

```diff
@@ -23,10 +23,10 @@ endif
 
 grimd: 
 	go get ./...
-	go build $(LDFLAGS) -o $(GOPATH)/bin/grimd ./grimd
+	go install $(LDFLAGS) github.com/MediaMath/grim/grimd
 
 tmp/grimd-$(VERSION).zip: grimd | tmp 
-	zip -r -j $@ $(GOPATH)/bin/grimd
+	export PATH=$$PATH:$$GOPATH/bin; zip -r -j $@ $$(which grimd)
```

Comment:
According to this: https://golang.org/cmd/go/#hdr-GOPATH_environment_variable it is,  "on Unix, the value is a colon-separated string. On Windows, the value is a semicolon-separated string. On Plan 9, the value is a list."